### ISSUE TITLE
Addon Validation Error

### ISSUE THREAD

**Issue by lunu-bounir** [mozilla/addons#908]
Please attach a copy of your add-on, and fill in the following information:

[sqlite-manager.zip](https://github.com/mozilla/addons/files/2860499/sqlite-manager.zip)

* Whether this error occurred during the submission of a new add-on, a new version of an existing add-on, a new file in an existing version, or an upload to the standalone validator.

a new version of an existing add-on

* Whether your add-on is listed or unlisted.

Listed

* The URL where you submitted the add-on.

https://addons.mozilla.org/en-US/developers/addon/sqlite-manager-webext/versions/submit/

**Comment by lunu-bounir** [comment]
```Error: Validation was unable to complete successfully due to an unexpected error.```

**Comment by wagnerand** [comment]
Running that file against the linter locally outputs:

```
    0: ExitFrame [pc: 0x2dadb684fc7d]
Security context: 0x20ee4c59d949 <JSObject>
    1: makeNext [0x20ee3a1e62b9] [/usr/local/lib/node_modules/addons-linter/node_modules/eslint/lib/code-path-analysis/fork-context.js:~150] [pc=0x2dadb6f16a7a](this=0x20ee62d96231 <ForkContext map = 0x20ee6a365cb1>,-1,-1)
    2: forkPath [0x20ee3a1e5b11] [/usr/local/lib/node_modules/addons-linter/node_modules/eslint/lib/code-path-analysis/code-path-state.js...

FATAL ERROR: Ineffective mark-compacts near heap limit Allocation failed - JavaScript heap out of memory
```

😳

**Comment by lunu-bounir** [comment]
Could it be due to the included libraries? Any idea how to fix this? As far as I know, there is no new library in the current version. I only updated the math.js one.

https://github.com/lunu-bounir/sqlite-manager/tree/master/data/manager/vendor

**Comment by lunu-bounir** [comment]
Okay I can reproduce this on a basic addon with just `sql.js` included (https://github.com/kripken/sql.js/tree/master/js)

[untitled.zip](https://github.com/mozilla/addons/files/2860682/untitled.zip)

**Comment by EnTeQuAk** [comment]
Indeed, I assumed it may have been the `sql.js` given it has been compiled by emscripten. I haven't digged any deeper to see why that happens in detail.

Do you have build-steps to reproduce that file for me locally?

**Comment by lunu-bounir** [comment]
I am using https://github.com/kripken/sql.js/blob/master/js/sql.js

The makefile for this project is https://github.com/kripken/sql.js/blob/master/Makefile
